docs/content/1.getting-started/7.i18n/1.nuxt.md

```diff
@@ -0,0 +1,141 @@
+---
+navigation.title: Nuxt
+title: I18n in a Nuxt app
+description: 'Nuxt UI supports language Internationalization (i18n) of its components in your Nuxt application.'
+select:
+  items:
+    - label: Nuxt
+      icon: i-logos-nuxt-icon
+      to: /getting-started/i18n/nuxt
+    - label: Vue
+      icon: i-logos-vue
+      to: /getting-started/i18n/vue
+---
+
+## Usage
+
+Nuxt UI provides a Vue component [UApp](/components/app) for globally configuring locale and other settings.
+
+```vue [app.vue]
+<script setup lang="ts">
+import { de } from '#ui/locale'
+</script>
+
+<template>
+  <UApp :locale="de">
+    <NuxtPage />
+  </UApp>
+</template>
+```
+
+::tip
+Nuxt UI components are using English **by default**.
+::
+
+### Dynamic locale
+
+If you need to switch between languages, you can use the `UApp` component with `locale` prop.
+
+We rely on [NuxtI18n](https://i18n.nuxtjs.org//) to manage different translations we want to support.
+
+::code-group{sync="pm"}
+```bash [pnpm]
+pnpm add @nuxtjs/i18n@next
+```
+
+```bash [yarn]
+yarn add @nuxtjs/i18n@next
+```
+
+```bash [npm]
+npm install @nuxtjs/i18n@next
+```
+
+```bash [bun]
+bun add @nuxtjs/i18n@next
+```
+::
+
+Go ahead and add some languages at `nuxt.config.ts`
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  i18n: {
+    locales: [
+      {
+        code: 'en',
+        name: 'English'
+      },
+      {
+        code: 'de',
+        name: 'Deutsch'
+      }
+    ]
+  }
+})
+````
+
+And then configure get `locale` and pass a dynamic `appLocale`.
+
+```vue [app.vue]
+<script setup lang="ts">
+import { en, de } from '#ui/locale'
+
+const { locale } = useI18n()
+const appLocale = computed(() => locale.value === 'de' ? de : en)  
+</script>
+
+<template>
+  <UApp :locale="appLocale">
+    <NuxtPage />
+  </UApp>
+</template>
+```
+
+### Dynamic direction
+
+To dynamically change the direction of components, we could leverage the [useTextDirection](https://vueuse.org/core/useTextDirection/) composable and combine it with our `UApp`.
+
+But first, we need to install the [@vueuse/core](https://vueuse.org/) package.
```

Comment:
`@vueuse/core` is already a dependency of `@nuxt/ui`, do we really need to specify this?

Reply:
It's really a dual situation. On the one hand, we can use internal dependencies of other packages. But if that package suddenly removes that dependency internally, then everything in our application will break.

I advocate that if you are going to use a package, you should explicitly install it